dotCMS/src/main/java/com/dotcms/util/pagination/CategoriesPaginator.java

```diff
@@ -0,0 +1,59 @@
+package com.dotcms.util.pagination;
+
+import java.util.Collection;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)